bp_common/syn/vivado/tcl/vivado_synth.tcl

```diff
@@ -0,0 +1,111 @@
+set proj_name "blackparrot_test"
+
+# Genesys 2 - xc7k325tffg900-2
+# Artix 7 - xc7a200tfbg676-2
+create_project -force $proj_name ./$proj_name -part $::env(PART)
```

Comment:
we want to run in non-project mode for speed and control:

http://grittyengineer.com/vivado-non-project-mode-releasing-vivados-true-potential/

Reply:
https://hwjedi.wordpress.com/2017/01/04/vivado-non-project-mode-the-only-way-to-go-for-serious-fpga-designers/